primitives/src/opcodes.rs

```diff
@@ -424,7 +424,7 @@ impl Opcode {
     ///
     /// Returns `None` if `self` is not a PUSHNUM.
     #[inline]
-    pub(crate) const fn decode_pushnum(self) -> Option<u8> {
+    pub const fn decode_pushnum(self) -> Option<u8> {
```

Comment:
I don't like publishing some obscure, badly documented function. I think we should either move it together with scripts or keep the function in the other crate for now.

Reply:
Fair, i'll re-spin. Thanks man.

Reply:
I had a quick look, this is not going to be released, can we just add a issue to make it private again before release?

Reply:
That's exactly what I'm afraid of: that we will forget anyway. Maybe a milestone could help.

Reply:
Added https://github.com/rust-bitcoin/rust-bitcoin/issues/2992 and a milestone for the `primitives` release.